<!DOCTYPE html PUBLIC"-//W3C//DTD XHTML 1.0 Transitional//EN""http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type"content="text/html; charset=utf-8"/><title>日志信息</title><link rel="stylesheet" href="../css.css"/></head><body>[Log]<p>记录时间：2019/1/22 17:17:27</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:07:45</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:05:14</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:04:37</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:03:59</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:03:24</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:03:20</p><p>日志类型：错误信息</p><p>日志描述：Object reference not set to an instance of an object.</p><p>异常根源：BluePI.Service</p><p>堆栈轨迹：   </br>at BluePI.Service.BaseService`1.GetById(Int32 id) in E:\Project\BluePI\BluePI.Service\BluePI.Service\BaseService.cs:line 71
   </br>at BluePI.Controllers.AdminController.GetUser() in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 73
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:02:25</p><p>日志类型：错误信息</p><p>日志描述：Multiple actions matched. The following actions matched route data and had all constraints satisfied:

BluePI.Controllers.AdminController.LogOn (BluePI)
BluePI.Controllers.AdminController.GetUser (BluePI)</p><p>异常根源：Microsoft.AspNetCore.Mvc.Core</p><p>堆栈轨迹：   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionSelector.SelectBestCandidate(RouteContext context, IReadOnlyList`1 candidates)
   </br>at Microsoft.AspNetCore.Mvc.Internal.MvcAttributeRouteHandler.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.Tree.TreeRouter.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.RouteCollection.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:01:53</p><p>日志类型：错误信息</p><p>日志描述：Multiple actions matched. The following actions matched route data and had all constraints satisfied:

BluePI.Controllers.AdminController.LogOn (BluePI)
BluePI.Controllers.AdminController.GetUser (BluePI)</p><p>异常根源：Microsoft.AspNetCore.Mvc.Core</p><p>堆栈轨迹：   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionSelector.SelectBestCandidate(RouteContext context, IReadOnlyList`1 candidates)
   </br>at Microsoft.AspNetCore.Mvc.Internal.MvcAttributeRouteHandler.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.Tree.TreeRouter.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.RouteCollection.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:01:27</p><p>日志类型：错误信息</p><p>日志描述：Multiple actions matched. The following actions matched route data and had all constraints satisfied:

BluePI.Controllers.AdminController.LogOn (BluePI)
BluePI.Controllers.AdminController.GetUser (BluePI)</p><p>异常根源：Microsoft.AspNetCore.Mvc.Core</p><p>堆栈轨迹：   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionSelector.SelectBestCandidate(RouteContext context, IReadOnlyList`1 candidates)
   </br>at Microsoft.AspNetCore.Mvc.Internal.MvcAttributeRouteHandler.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.Tree.TreeRouter.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.RouteCollection.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:01:23</p><p>日志类型：错误信息</p><p>日志描述：Multiple actions matched. The following actions matched route data and had all constraints satisfied:

BluePI.Controllers.AdminController.LogOn (BluePI)
BluePI.Controllers.AdminController.GetUser (BluePI)</p><p>异常根源：Microsoft.AspNetCore.Mvc.Core</p><p>堆栈轨迹：   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionSelector.SelectBestCandidate(RouteContext context, IReadOnlyList`1 candidates)
   </br>at Microsoft.AspNetCore.Mvc.Internal.MvcAttributeRouteHandler.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.Tree.TreeRouter.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.RouteCollection.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 17:01:17</p><p>日志类型：错误信息</p><p>日志描述：Multiple actions matched. The following actions matched route data and had all constraints satisfied:

BluePI.Controllers.AdminController.LogOn (BluePI)
BluePI.Controllers.AdminController.GetUser (BluePI)</p><p>异常根源：Microsoft.AspNetCore.Mvc.Core</p><p>堆栈轨迹：   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionSelector.SelectBestCandidate(RouteContext context, IReadOnlyList`1 candidates)
   </br>at Microsoft.AspNetCore.Mvc.Internal.MvcAttributeRouteHandler.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.Tree.TreeRouter.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Routing.RouteCollection.RouteAsync(RouteContext context)
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /><p>记录时间：2019/1/22 10:52:54</p><p>日志类型：错误信息</p><p>日志描述：String reference not set to an instance of a String.
Parameter name: s</p><p>异常根源：System.Private.CoreLib</p><p>堆栈轨迹：   </br>at System.Text.Encoding.GetBytes(String s)
   </br>at BluePI.Helper.PwdHelper.Get32MD5Two(String source) in E:\Project\BluePI\BluePI.Helper\PwdHelper.cs:line 64
   </br>at BluePI.Repository.UserRepository.LogOn(User user) in E:\Project\BluePI\BluePI.Repository\BluePI.Repository\UserRepository.cs:line 24
   </br>at BluePI.Service.UserService.LogOn(User user) in E:\Project\BluePI\BluePI.Service\BluePI.Service\UserService.cs:line 34
   </br>at BluePI.Controllers.AdminController.LogOn(logOnModel logOnModel) in E:\Project\BluePI\BluePI\Controllers\AdminController.cs:line 53
   </br>at lambda_method(Closure , Object , Object[] )
   </br>at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   </br>at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   </br>at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   </br>at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   </br>at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   </br>at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   </br>at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   </br>at BluePI.ExceptionHelp.ExceptionFilter.Invoke(HttpContext context) in E:\Project\BluePI\BluePI\ExceptionHelp\ExceptionFilter.cs:line 39</p><hr /></body></html>